csrc/scheduler/mma_utils.cpp

```diff
@@ -837,6 +837,117 @@ void WarpMmaSwizzler::scheduleLdMatrix(TensorView* tv, MmaOperand operand) {
   setWarpMapped(tv, 2);
 }
 
+void WarpMmaSwizzler::parallelizeAsBulkSkippingFirstIDs(
+    TensorView* tv,
+    size_t first_ids_to_skip) {
```

Comment:
nit: should we always use `int64_t` for integers in all our functions? clang-tidy loves to complain about mismatching integer types and using the same integer type could help avoiding having to add casts between types.